src/Traits/CacheModelQueries.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+namespace FluxErp\Traits;
+
+use FluxErp\Support\Database\CachedBuilder;
+use Illuminate\Database\Eloquent\Model;
+use Illuminate\Support\Facades\Cache;
+
+trait CacheModelQueries
+{
+    public static function bootCacheModelQueries(): void
+    {
+        static::$builder = CachedBuilder::class;
```

Comment:
how is this working? On app start i get an exception: access to undeclared static property.